### PR TITLE
Add installation guide for Linux

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
{insert GNU/Linux copypasta here}

### CODE REVIEW

**Comment by ZeroPyrozen** [comment]
Add missing line, list item and remove unnecessary `:` per [this action](https://github.com/ppy/osu-wiki/runs/4426962245?check_suite_focus=true) run

**Comment by peppy** [comment]
I dunno about this. I'd rather link to a third party source with installation instructions rather than make this something official.

File: wiki/Installation/Linux/en.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+pulse.min.req = 32/48000 # 0.67ms
+pulse.min.quantum = 32/48000 # 0.67ms
+}
+```
+
+4. Save with Ctrl+O and exit with Ctrl+X.
+5. Reboot your PC.
+6. Enable compatibility mode on osu!.
+
+### - PP Counter for OBS: gosumemory
```
Comment (peppy) [inline wiki/Installation/Linux/en.md:434]: This kind of stuff most *definitely* cannot be on the official wiki.

Comment (daud-ahn) [reply wiki/Installation/Linux/en.md:434]: if you are talking about those additional info, it can just be removed

Comment (peppy) [reply wiki/Installation/Linux/en.md:434]: No applications, links to executables, etc. can be in this guide.

I would propose this is just a stub leading to off-site guides to getting stuff working. I don't believe it should be present in the official wiki. Officially we do not support linux, and don't intend to offer support that isn't via osu!(lazer).

In fact I'd argue that this guide should be about osu!(lazer) instead.

Comment (daud-ahn) [reply wiki/Installation/Linux/en.md:434]: how about, move this page content to my gist (or maybe forum post) and then link that to this page.
about osu!lazer

Comment (peppy) [reply wiki/Installation/Linux/en.md:434]: As long as it doesn't link to custom osu/wine packages. It should be instructions on how to make things work with no external binaries.

**Comment by peppy** [comment]
Going to close this PR for now because I don't want the commits in git history regardless of how this is changed going forward. Feel free to open an issue to discuss this further.

**Pull request closed without merging**